build/android/bytecode/java/org/brave/bytecode/BraveAndroidSyncSettingsClassAdapter.java

```diff
@@ -15,28 +15,28 @@
 
     public BraveAndroidSyncSettingsClassAdapter(ClassVisitor visitor) {
         super(visitor);
```

Comment:
We don't probably need that file at all now

Reply:
addressed in aafc5e184ead94535d08123a21919a61bc30c2a1